.github/workflows/publish-bolt-crates.yml

```diff
@@ -205,6 +205,9 @@ jobs:
           cargo publish $DRY_RUN_FLAG --manifest-path=programs/bolt-component/Cargo.toml --token $CRATES_TOKEN $NO_VERIFY_FLAG
           cargo publish $DRY_RUN_FLAG --manifest-path=programs/world/Cargo.toml --token $CRATES_TOKEN $NO_VERIFY_FLAG
           cargo publish $DRY_RUN_FLAG --manifest-path=crates/bolt-lang/Cargo.toml --token $CRATES_TOKEN $NO_VERIFY_FLAG
+          if [ "${DRY_RUN}" != "true" ]; then
+            cargo publish --manifest-path=cli/Cargo.toml --token $CRATES_TOKEN
+          fi
```

Comment:
@GabrielePicco 
Unlike the other crates, I cannot run `cargo publish --manifest-path=cli/Cargo.toml --no-verify` because it fails with:
```
error: failed to prepare local package for uploading

Caused by:
  failed to select a version for the requirement `world = "=0.1.11"`
  candidate versions found which didn't match: 0.1.10, 0.1.9, 0.1.8, ...
  location searched: crates.io index
  required by package `bolt-cli v0.1.11 (/root/dev/magicblock-labs/bolt/cli)`
```

It happens because the crate isn't published yet. This verification is skipped on other crates, like `bolt-lang`, with `-no-verify`. But for some reason, it isn't working for `bolt-cli`.

Would it be probably because it's a binary crate? Idk

Reply:
Looks like it's searching on crates.io rather than resolving the world dependency from the local path